src/main/java/com/google/api/gax/grpc/BundlingCallSettings.java

```diff
@@ -43,21 +43,21 @@
 import java.util.concurrent.ScheduledExecutorService;
 
 /**
- * A settings class to configure an ApiCallable for calls to an API method that supports
+ * A settings class to configure an UnaryApiCallable for calls to an API method that supports
```

Comment:
s/an/a/

Do a search through all of the code to replace all of the cases. Note, there is at least one capitalized case (s/An/A).